keyboards/zvecr/split_blackpill/config.h

```diff
@@ -0,0 +1,191 @@
+/* Copyright 2020 zvecr<git@zvecr.com>
```

Comment:
Nitpicky, but this bugs me.

```suggestion
/* Copyright 2020 zvecr <git@zvecr.com>
```